templates/common/_base/files/NetworkManager-keyfiles.yaml

```diff
@@ -2,6 +2,8 @@ mode: 0644
 path: "/etc/NetworkManager/conf.d/99-keyfiles.conf"
 contents:
   inline: |
+    [main]
+    plugins=keyfile,ifcfg-rh
```

Comment:
why are we doing this while Bug 1928473: Apply system-connections-merged to all platforms already fixed the above ?

Reply:
No, I don't think it fixed this issue. On RHCOS, the default plugin is set explicitly with `plugins=keyfile,ifcfg-rh` in 10-disable-default-plugins.conf. So I think we need to do the same thing for RHEL7.